pkg/master/master_test.go

```diff
@@ -64,9 +64,12 @@ func setUp(t *testing.T) (Master, Config, *assert.Assertions) {
 	fakeClient := tools.NewFakeEtcdClient(t)
 	fakeClient.Machines = []string{"http://machine1:4001", "http://machine2", "http://machine3:4003"}
 	storageVersions := make(map[string]string)
-	config.DatabaseStorage = etcdstorage.NewEtcdStorage(fakeClient, testapi.Default.Codec(), etcdtest.PathPrefix())
+	databaseStorage := DatabaseClients{
+		Default:      etcdstorage.NewEtcdStorage(fakeClient, testapi.Default.Codec(), etcdtest.PathPrefix()),
```

Comment:
Oh, I see-- I forgot the codec is baked into the Storage object. I guess you actually need a per group default in the DatabaseClients object.